tests/e2e/authcontroller_test.go

```diff
@@ -100,8 +100,10 @@ func (tc *AuthControllerTestCtx) validateAuthCRDefaultContent() error {
 		return errors.New("AdminGroups is empty ")
 	}
 
-	if tc.testAuthInstance.Spec.AdminGroups[0] != dashboard.GetAdminGroup() {
-		return fmt.Errorf("expected %s, found %v", dashboard.GetAdminGroup(), tc.testAuthInstance.Spec.AdminGroups[0])
+	if tc.platform == cluster.SelfManagedRhoai || tc.platform == cluster.ManagedRhoai {
+		return fmt.Errorf("expected rhods-admins, found %v", tc.testAuthInstance.Spec.AdminGroups[0])
+	} else if tc.testAuthInstance.Spec.AdminGroups[0] != "odh-admins" {
+		return fmt.Errorf("expected odh-admins, found %v", tc.testAuthInstance.Spec.AdminGroups[0])
 	}
```

Comment:
i am not sure i understand this change:
so this is to test if the platform is for self-managed or managed, and error out?
but the test should not care about this,
instead (simiar to validateRemovingGroups):  
- if platform is cluter.OpenDataHub || cluster.Unknown=> odh-admins as AdminGroups[0]
- if is SelfManaged => rhods-admins as AdminGroups[0]
- if is Managed => dedicated-admins

Reply:
:+1:  

The actual tests were missing for some reason. I've remade the commit and pushed the amend, not sure why they weren't there :/